packages/abi-gen-templates/partials/callAsync.handlebars

```diff
@@ -24,6 +24,10 @@ async callAsync(
         },
         self._web3Wrapper.getContractDefaults(),
     );
+    if (callDataWithDefaults.from !== undefined) {
+        callDataWithDefaults.from = callDataWithDefaults.from.toLowerCase();
+    }
```

Comment:
Can we do these normalizations right after the assertions in each method? They should be as close to the top as possible.

Reply:
Oh, or is this here because `self._web3Wrapper.getContractDefaults()` might return checksummed `from`s?

Reply:
yup, the output of `self._web3Wrapper.getContractDefaults()` is passed in at instantiation and could be anything